src/common/entity/compute_state_display.ts

```diff
@@ -130,7 +130,7 @@ export const computeStateDisplayFromEntityAttributes = (
         date = new Date(attributes.year, attributes.month - 1, attributes.day);
         return formatDate(date, locale);
       }
-      if (attributes.has_time) {
+      if (domain === "time" || attributes.has_time) {
         date = new Date();
         date.setHours(attributes.hour, attributes.minute);
```

Comment:
The `time` entity doesn't have these attributes. The above time parsing should already cover it, right?